Content.Server/Sericulture/SericultureSystem.cs

```diff
@@ -1,22 +1,15 @@
-using Content.Server.Actions;
-using Content.Server.DoAfter;
-using Content.Server.Popups;
-using Content.Shared.Actions;
-using Content.Shared.Actions.ActionTypes;
-using Content.Shared.DoAfter;
-using Content.Shared.Nutrition.Components;
-using Content.Shared.Nutrition.EntitySystems;
-using Robust.Shared.Prototypes;
 using Content.Shared.Sericulture;
+using Content.Server.Popups;
 using Content.Server.Stack;
+using Content.Shared.Nutrition.EntitySystems;
 
 namespace Content.Server.Sericulture;
 
-public sealed class SericultureSystem : EntitySystem
+/// <summary>
+/// <inheritdoc/>.
+/// </summary>
+public sealed partial class SericultureSystem : SharedSericultureSystem
```

Comment:
you dont have a clientside system inheriting shared one so none of the shared code is getting executed on the client

Reply:
So I should have something like this?

![image](https://github.com/space-wizards/space-station-14/assets/85175107/a8cd7f73-e146-409c-a4e8-8faad7ebb751)

Reply:
yes